src/components/operations/operation-details/ko/runtime/apiKeyDetails.ts

```diff
@@ -0,0 +1,9 @@
+export enum ApiKeyLocation {
+    Query = "query",
+    Header = "header"
+}
+
+export interface ApiKeyDetails {
```

Comment:
Instead of ApiKey, AccessToken would be more accurate I think. Because It refers to here I guess:
![image](https://user-images.githubusercontent.com/26141403/207020711-fa563922-f4ed-4b37-bd41-3e963fadc47a.png)

Reply:
I used ApiKey to respect the OpenAPI specifications: https://swagger.io/docs/specification/authentication/
Since the idea of this feature is to display the security schemes as they are in OpenAPI, I think it makes more sense to use the same naming convention as they do.

I guess AccessToken make sense, but it will still be mapped to `API Key` in the Security Schemes section, which, I think, would be confusing. 

What do you think?

Reply:
yes it makes sense then, thanks!